source/includes/_data-dictionary.md.erb

```diff
@@ -37,8 +39,6 @@ The following list of fields are posted annually, per reporting year.
 | property_type_code   	     | character varying (255) | <%= dictionary_description(:property_type_code) %>   	    |			                 |
 | asset_gav                  | numeric                 | <%= dictionary_description(:asset_gav) %>                  | currency (millions)  |
 | asset_size                 | numeric                 | <%= dictionary_description(:asset_size) %>                 | m<sup>2</sup>/sq. ft |
-| energy_rating_id           | integer                 | <%= dictionary_description(:energy_rating_id) %>           |                      |
-| energy_rating_size         | numeric                 | <%= dictionary_description(:energy_rating_size) %>         | m<sup>2</sup>/sq. ft |
```

Comment:
description can be removed from https://github.com/GRESB/api-docs/blob/master/data/dictionary.yml#L168